src/components/ProfileAvatar.tsx

```diff
@@ -0,0 +1,229 @@
+import Skeleton from '@mui/material/Skeleton'
+import Slider from '@mui/material/Slider'
+import VerifiedBorder from 'assets/VerifiedProfilePic.svg'
+import React, { useEffect, useState } from 'react'
+import { Button } from '@mui/material'
+import { Modal } from 'react-bootstrap'
+import Cropper from 'react-easy-crop'
+import { Area } from 'react-easy-crop/types'
+import { SynapseConstants } from 'synapse-react-client'
+import { displayToast } from 'synapse-react-client/dist/containers/ToastMessage'
+import {
+  getFileHandleByIdURL,
+  getMyUserBundle,
+  updateMyUserProfile,
+  uploadFile,
+} from 'synapse-react-client/dist/utils/SynapseClient'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import {
+  FileUploadComplete,
+  UserBundle,
+} from 'synapse-react-client/dist/utils/synapseTypes'
+import { getCroppedImg } from './CropImage'
+
+export type ProfileAvatarProps = {}
+
+export const ProfileAvatar = (props: ProfileAvatarProps) => {
+  const { accessToken } = useSynapseContext()
+
+  const [userBundle, setUserBundle] = useState<UserBundle>()
+  const [profilePicUrl, setProfilePicUrl] = useState<string | undefined>()
+  const [verified, setVerfied] = useState<boolean>()
+  const [isLoading, setIsLoading] = useState(true)
+  const [image, setImage] = useState<string | undefined>()
+  const [croppedArea, setCroppedArea] = useState<Area | undefined>()
+  const [crop, setCrop] = useState({ x: 0, y: 0 })
+  const [zoom, setZoom] = useState(1)
+  const [openCropModal, setOpenCropModal] = useState(false)
+  const [imageLoading, setImageLoading] = useState(false)
+
+  const getProfile = async () => {
+    setIsLoading(true)
+    try {
+      const mask =
+        SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED |
+        SynapseConstants.USER_BUNDLE_MASK_USER_PROFILE
+
+      const bundle: UserBundle = await getMyUserBundle(mask, accessToken)
+      let picUrl
+      if (bundle.userProfile?.profilePicureFileHandleId) {
+        picUrl = await getFileHandleByIdURL(
+          bundle.userProfile?.profilePicureFileHandleId as string,
+          accessToken,
+        )
+      }
+      setUserBundle(bundle)
+      setVerfied(bundle.isVerified)
+      setProfilePicUrl(picUrl)
+    } catch (err: any) {
+      displayToast(err.reason as string, 'danger')
+    } finally {
+      setIsLoading(false)
+    }
+  }
+
+  const updateUserProfile = async (newFileHandleId: string) => {
+    try {
+      if (userBundle?.userProfile) {
+        userBundle.userProfile.profilePicureFileHandleId =
+          newFileHandleId as string
+        await updateMyUserProfile(userBundle.userProfile, accessToken)
+        displayToast('Profile picture has been successfully updated', 'success')
+        getProfile()
```

Comment:
In the old Profile page implementation, you could update your picture without updating your profile.  This version updates immediately.